
Been gettign the following error until the removing the mount line

```
Starting django ... error

ERROR: for django  Cannot start service django: error while creating mount source path '/z1-rtkushner1/rtkushner1/riboxyz/ribxz': mkdir /z1-rtkushner1/rtkushner1/riboxyz/ribxz: permission denied

ERROR: for django  Cannot start service django: error while creating mount source path '/z1-rtkushner1/rtkushner1/riboxyz/ribxz': mkdir /z1-rtkushner1/rtkushner1/riboxyz/ribxz: permission denied
ERROR: Encountered errors while bringing up the project.
```